modules/steam/testbeds/steam.nix

```
@@ -0,0 +1,35 @@
+{
+  config,
+  lib,
+  pkgs,
+  ...
+}:
+
+{
+  stylix.testbed = {
+    enable = pkgs.stdenv.hostPlatform.system == "x86_64-linux";
+    ui.command = {
+      text = "steam";
+      useTerminal = true;
+    };
+    ui.application = {
+      name = "io.github.Foldex.AdwSteamGtk";
+      package = pkgs.adwsteamgtk;
+    };
+  };
+
+  virtualisation.vmVariant.virtualisation = {
+    cores = 4;
+    memorySize = 2 * 1024;
+    diskSize = 3 * 1024;
+  };
```

> The default amount of 2GB is *just quite* not enough to let the client update and start.

The other default re-declarations can be removed:

```suggestion
  virtualisation.vmVariant.virtualisation.diskSize = 3 * 1024;
```